src/main/java/com/yacht/molvenorestaurant/business/MenuManager.java

```diff
@@ -0,0 +1,34 @@
+package com.yacht.molvenorestaurant.business;
+
+import com.yacht.molvenorestaurant.model.MenuCategory;
+
+import java.util.ArrayList;
+
+public class MenuManager {
+
+    public ArrayList<MenuCategory> getCategoriesList() {
+        return categoriesList;
+    }
+
+    public void setCategoriesList(ArrayList<MenuCategory> categoriesList) {
+        this.categoriesList = categoriesList;
+    }
+
+    private ArrayList<MenuCategory> categoriesList;
```

Comment:
gebruik dishList ipv categoriesList in de manager. Dan kan je deze list weghalen